custom_components/tuya_local/devices/aubess_rainpoint_irrigation_system.yaml

```diff
@@ -0,0 +1,140 @@
+name: Aubess Irrigation System 
+products:
+  - id: 2ak7r2culspkc7hx
+    model: RainPoint TTP106W
+    manufacturer: Aubess
+primary_entity:
+  entity: switch
+  icon: "mdi:pipe-valve"
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+
+secondary_entities:
+  - entity: sensor
+    name: Work State
```

Comment:
The same comment about naming convention applies here, but also "Work state" is just following the Tuya protocol naming, normally these will be labelled as something else in the app.  Maybe just "Status"?

The values below should be how they should appear in the UI (human readable) so generally capitalised and with spaces rather than underscores, since there is currently no translation done for enum sensor values.